docker-compose.yml

```
@@ -30,6 +30,10 @@ services:
 
   traefik:
     image: traefik:v3.4.4
+    networks:
+      opencloud-net:
+        aliases:
+          - host.docker.internal
```

This needs to be kept in sync with the `OC_URL`.

```suggestion
          - host.docker.internal:9200
```